app/src/components/account/UserMenu.vue

```diff
@@ -196,6 +283,47 @@ export default {
     getAddressNetwork(address) {
       return this.networks.find((network) => network.id === address.networkId)
     },
+    getSessionName(session) {
+      switch (session.sessionType) {
+        case `local`:
+          return session.name
+        case `extension`:
+          return session.name
+        default:
+          return session.sessionType
+            .charAt(0)
+            .toUpperCase()
+            .concat(session.sessionType.slice(1))
+      }
+    },
+    pollForNewExtensionAddresses() {
```

Comment:
this seems like overkill. we're going to poll every 30s for every user even if they don't have the extension installed?

what if the extension tells lunie that there's a new message like we do when a transaction is signed?

Reply:
Ah OK, found it. Working on having a new message type right now. We need this prior to moving anything to session

Reply:
@faboweb can you help me out on this one? Look at my last commit please. I am sending the message from extension (`NEW_ADDRESS`) but somehow only the `INIT_EXTENSION` message works. The other ones the app just won't react.

Thanks :pray:

Reply:
OK, we will just do this in another PR. For now I removed the whole auto-update for extension accounts